plenum/test/replica/stashing/test_replica_unstashing.py

```diff
@@ -15,7 +15,6 @@ def msg(replica):
 
 def test_unstash_catchup(replica, msg):
     pre_prepare, _ = msg
-    replica.set_view_no(pre_prepare.viewNo + 1)
```

Comment:
Why remove this?

Reply:
Because we do not need to change the view_no.
Fix for this comment - https://github.com/hyperledger/indy-plenum/pull/1280#discussion_r313899506